ci-operator/jobs/shiftstack/merge-bot/shiftstack-merge-bot-main-periodics.yaml

```diff
@@ -52,6 +52,59 @@ periodics:
     - name: result-aggregator
       secret:
         secretName: result-aggregator
+- agent: kubernetes
+  cluster: build02
+  cron: 0 12 * * Mon,Thu
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  extra_refs:
+  - base_ref: main
+    org: shiftstack
+    repo: merge-bot
+  labels:
+    ci.openshift.io/generator: prowgen
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-shiftstack-merge-bot-main-cluster-api-provider-openstack
```

Comment:
do we already have a slack notification by `merge-bot`? If not we need to add it here.

Reply:
we do, for example the rehearsal sent a message on slack.